docs/backend/workflows.md

```diff
@@ -0,0 +1,19 @@
+## Workflows
+
+Aspen has a number of long-lived workflows that are run through AWS Batch.  This document explains each of these workflows and how to manually invoke them.
+
+### Ingest GISAID
+
+This workflow downloads a [GISAID hCov-19 data dump](https://www.gisaid.org) and records a [`RawGisaidDump`](https://github.com/chanzuckerberg/aspen/search?q=%22class+RawGisaidDump%22) object referencing the dump.  This workflow can be manually invoked using the AWS cli: `aws batch submit-job --job-name 'ingest-gisaid' --job-queue aspen-batch --job-definition aspen-batch-job-definition --container-overrides '{"command": ["trunk", "src/py/workflows/ingest_gisaid/ingest.sh"]}'`
+
+This workflow automatically invokes the [transform-gisaid workflow](#transform-gisaid) upon completion, using the [`RawGisaidDump`](https://github.com/chanzuckerberg/aspen/search?q=%22class+RawGisaidDump%22) as input.
+
+### Transform GISAID
+
+This workflow applies the [transform-gisaid script](https://github.com/nextstrain/ncov-ingest/blob/master/bin/transform-gisaid) to transform the data into an input format suitable for the nextstrain pipeline.  The input for this workflow is a [`RawGisaidDump`](https://github.com/chanzuckerberg/aspen/search?q=%22class+RawGisaidDump%22) object and records a [`ProcessedGisaidDump`](https://github.com/chanzuckerberg/aspen/search?q=%22class+ProcessedGisaidDump%22) object referencing the processed dump.  A [`GisaidDumpWorkflow`](https://github.com/chanzuckerberg/aspen/search?q=%22class+GisaidDumpWorkflow%22) is used to link the input to the output.
```

Comment:
can you expand a bit more on what suitable means here?

Reply:
I think it is just the metadata format.  I will clarify.